### PR TITLE
Added JSON functionality to JavaScript & Servlet

### PR DESCRIPTION
**Pull request by kristen-m**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -53,8 +53,10 @@ function fixSite() {
 }
 
 
-function getGreeting() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('greeting-container').innerText = quote;
+function getFruit() {
+  fetch('/data').then(response => response.json()).then((fruit) => {
+    document.getElementById('fruit-container').innerText = fruit;
```
Comment (damargulis) [inline portfolio/src/main/webapp/script.js]: Does this work?
the fruit variable here is an object right, not a string -- so what does this look like when its run?

Comment (kristen-m) [reply portfolio/src/main/webapp/script.js]: This is what it looks like when I run this https://screenshot.googleplex.com/9T90jXoJhbj

In retrospect, naming it fruit there might be confusing, it's really just the text that's being retrieved by the json() call I think (which in this case happens to be a list of fruits)

Comment (damargulis) [reply portfolio/src/main/webapp/script.js]: Ah I see

So what's happening here is that the response is the string
"["blueberry", "Cherry", ...]
This is getting parsed into an Array javascript object. It looks like when you set the textContent of a div to an array, the div will automatically join those values into a comma separated string.
If you want to make it look just a bit better, you can set innerText to `fruit.join(', ');` which will include a space between the names.

What you have is also fine -- just make sure you understand whats happening here -- you're using an object here, not just a string

**Review by damargulis — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (kristen-m) [reply portfolio/src/main/webapp/script.js]: Got it!

**Pull request merged**